src/Toolkit.Preview/Toolkit.Preview/UI/TableOfContents/TocItem.cs

```diff
@@ -59,6 +59,7 @@ internal TocItem(object content, bool showLegend, int depth, TocItem? parent)
                 if (content is ILoadable loadable && loadable.LoadStatus != LoadStatus.Loaded)
                 {
                     loadable.Loaded += (s, e) => SetChildren();
+                    _ = loadable.LoadAsync();
```

Comment:
I don't think we should be doing this explicitly. The mapview should be auto-loading the layers.